src/config/browser-options.js

```diff
@@ -31,9 +32,21 @@ exports.getPerBrowser = () => {
             validate: (value, config) => {
                 if (_.isNull(value) && _.isNull(config.desiredCapabilities)) {
                     throw new Error('Each browser must have "desiredCapabilities" option');
-                } else {
-                    utils.assertOptionalObject(value, "desiredCapabilities");
                 }
+
+                const { browserName, browserVersion, webSocketUrl } = value;
+
+                if (webSocketUrl === true && browserName === "chrome" && browserVersion) {
```

Comment:
user may not specify option `browserVersion`, in this case will be used the latest version of chrome (for example with `gridUrl: local`)

Reply:
what about another browsers? firefox, for example

Reply:
ok, I support it in firefox too